project.py

```diff
@@ -0,0 +1,174 @@
+"""
+Mini Project 5--Re-editing of Mini Project 3 (Text Mining)
+Author: Nicholas Sherman
+Date: 4/22/2017
+Class: Software Design
+
+This project markov chains Family Guy scripts with Donald Trump speeches for entertainment purposes.
+"""
+from bs4 import BeautifulSoup
+import requests
+import re
+import pickle
+import os.path
+from pathlib import Path
+import random
+
+def dict_creation_one(script, things = dict()):
+    '''Create the base dictionary for ONE word; only to be called if there is no dicts1.p already.
```

Comment:
I appreciate the new header comments and file documentation - the program itself is cleaner and more follow-able as a result.